common/src/main/resources/Operator.properties

```diff
@@ -238,6 +238,7 @@ WLSDO-0044=One or more server pods that are supposed to be available did not sta
   Check the server status in the domain status, the server pod status and logs, and WebLogic Server logs for possible reasons. \
   One common cause of this issue is a problem pulling the WebLogicServer image. \
   Adjust the value of 'serverPod.maxPendingWaitTimeSeconds' setting if needed."
+WLSDO-0045=Cannot reference Cluster ''{0}'' because it is used by ''{1}''
```

Comment:
should we say "Cluster resource" here to avoid possible confusion with WLS cluster?

Reply:
good point. will do.